test/gcs/main_test.go

```diff
@@ -177,10 +167,10 @@ func getHost(_ context.Context, tb testing.TB, rt runtime.Runtime) *hcsv2.Host {
 
 func getHostErr(rt runtime.Runtime, tp transport.Transport) (*hcsv2.Host, error) {
 	h := hcsv2.NewHost(rt, tp, &securitypolicy.ClosedDoorSecurityPolicyEnforcer{}, os.Stdout)
```

Comment:
why do we use the closed door policy here?

Reply:
I think originally the gcs started with it, so i copied it for consistency?
Ill change it to open door as well, since that matches the default of what gcs does now